### PR TITLE
[Modular] [Ready] Wake the fuck up, samurai. (Removes robohand from the uplink)

### PR DESCRIPTION
**Pull request by lyricalpaws**
## About The Pull Request
You'd think we wouldn't give traitors a cybernetic lockeded, 60 damage, insane delimb chance gun, with good armour. But we did

## Why It's Good For The Game
Many people asked for this. It's finally time we put Johnny Silverhand to bed

## Changelog
:cl:
tweak: Removes robohand from uplink (but not code)
qol: So long, space cowboy
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -362,13 +362,13 @@
...
-	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/robohand
-	cost = 35
-
-/datum/uplink_item/loadout_skyrat/robohand/purchase(mob/user, datum/component/uplink/U)
-	. = ..()
-	notify_ghosts(message = "[user] has purchased the Johnny Robohand bundle, watch him be a badass!", ghost_sound = 'modular_skyrat/modules/3516/sound/wakeup.ogg', source = user) //Everyone needs to know he's a badass
+///datum/uplink_item/loadout_skyrat/robohand
+//	name = "Robohand Bundle"
+//	desc = "Themed after the infamous terrorist(or not), Johnny Robohand. You have no reason to fail your objectives with this kit. The gun inside requires your arm to be robotic. \
+//			It comes with a robotic replacement arm. Wake the fuck up, samurai."
```
Comment (ZephyrTFA) [inline modular_skyrat/master_files/code/modules/uplink/uplink_items.dm]:
```suggestion
//	desc = "Themed after the infamous terrorist(or not), Johnny Robohand. You have no reason to fail your objectives with this kit. The gun inside requires your arm to be robotic. It comes with a robotic replacement arm. Wake the fuck up, samurai."
```

Comment (ZephyrTFA) [reply modular_skyrat/master_files/code/modules/uplink/uplink_items.dm]: Not sure why this description was originally two-line but it shouldn't.
Yes I know you are commenting it out, should still fix it anyway

**Comment by ghost** [comment]
Should have been titled 'Go back to sleep, Samurai' smh smh

**Comment by BrotherHangyul** [comment]
Go the fuck back to sleep, Samurai, it's Saturday morning and you don't have work tomorrow.

**Comment by lyricalpaws** [comment]
> 
> 
> Go the fuck back to sleep, Samurai, it's Saturday morning and you don't have work tomorrow.

Amazing reply

**Comment by Funce** [comment]
@tf-4 Balance change, input?

**Review by Funce — APPROVED** [review]
No more fancy uplink. Remains adminspawnable

**Pull request merged**